sqrl.go

```diff
@@ -133,3 +133,42 @@ func QueryRowWithContext(ctx context.Context, db QueryRowerContext, s Sqlizer) R
 	query, args, err := s.ToSql()
 	return &Row{RowScanner: db.QueryRowContext(ctx, query, args...), err: err}
 }
+
+// DBRunner wraps sql.DB to implement Runner.
+type dbRunner struct {
+	*sql.DB
+}
+
+func (r *dbRunner) QueryRow(query string, args ...interface{}) RowScanner {
+	return r.DB.QueryRow(query, args...)
+}
+
+func (r *dbRunner) QueryRowContext(ctx context.Context, query string, args ...interface{}) RowScanner {
+	return r.DB.QueryRowContext(ctx, query, args...)
+}
+
+// TxRunner wraps sql.Tx to implement Runner.
+type txRunner struct {
+	*sql.Tx
+}
+
+func (r *txRunner) QueryRow(query string, args ...interface{}) RowScanner {
+	return r.Tx.QueryRow(query, args...)
+}
+
+func (r *txRunner) QueryRowContext(ctx context.Context, query string, args ...interface{}) RowScanner {
+	return r.Tx.QueryRowContext(ctx, query, args...)
+}
+
+// WrapRunner returns Runner that implements QueryRower and QueryRowerContext.
+func wrapRunner(baseRunner BaseRunner) (runner Runner) {
```

Comment:
`baseRunner` may not be convertible to `Runner` interface (you never know which wrapper someone's going to use). Here's a test case:
```go
type mockBaseRunner struct{}

func (m *mockBaseRunner) Exec(query string, args ...interface{}) (sql.Result, error) { return nil, nil }
func (m *mockBaseRunner) ExecContext(ctx context.Context, query string, args ...interface{}) (sql.Result, error) {
	return nil, nil
}

func (m *mockBaseRunner) Query(query string, args ...interface{}) (*sql.Rows, error) { return nil, nil }
func (m *mockBaseRunner) QueryContext(ctx context.Context, query string, args ...interface{}) (*sql.Rows, error) {
	return nil, nil
}

func TestWrapRunner(t *testing.T) {
	runner := &mockBaseRunner{}
	wrap := wrapRunner(runner)
	assert.NotNil(t, wrap)
}
```
The test will fail, because `wrapRunner` returns `nil`. Perhaps, it would be better to change return type from `Runner` to `BaseRunner` and introduce a default case where `baseRunner` cannot be converted and should be returned as is.